tools/autograd/derivatives.yaml

```diff
@@ -1072,6 +1072,7 @@
 - name: mv(Tensor self, Tensor vec) -> Tensor
   self: grad.ger(vec.conj())
   vec: self.conj().t().mv(grad)
+  result: mv(self_t, vec_p) + mv(self_p, vec_t)
```

Comment:
it's not necessary, but I think it would be cool to add a ZeroTensor fastpath for `mv` as a follow-up.

Reply:
Yes!